client/controller/models/apps/apps.go

```diff
@@ -92,7 +92,7 @@ func Logs(c *client.Client, appID string, lines int) (string, error) {
 	u := fmt.Sprintf("/v2/apps/%s/logs", appID)
 
 	if lines > 0 {
-		u += "?log_lines=" + strconv.Itoa(lines)
+		u += "/?log_lines=" + strconv.Itoa(lines)
```

Comment:
Sorry... I read this wrong.  This is in client code.  Maybe this has to do with recent Django updates?